.travis.yml

```diff
@@ -7,17 +7,5 @@ before_install:
 install:
 - npm install
 script:
-- 'if [ $TRAVIS_BRANCH === "master" && $svn_pass ]; then
-    travis_wait 30 npm run-script build;
-    cd ..;
-    svn checkout --non-interactive --no-auth-cache --username=$svn_username --password=$svn_pass https://svn.apache.org/repos/asf/cordova/site cordova-website;
-    cp -R cordova-docs/build-prod/. cordova-website/public/;
-    cd cordova-website;
-    svn status | grep "?";
-    svn add --force .;
-    svn --non-interactive --no-auth-cache --username=$svn_username --password=$svn_pass commit -m "Updated docs";
-    cd ../cordova-docs;
-    npm test;
-  else
-    npm test;
-  fi'
\ No newline at end of file
+  - chmod 755 buildAndDeploy.sh
+  - travis_wait 120 sleep infinity & ./buildAndDeploy.sh
```

Comment:
might want to change the 120 to 30 i think

Reply:
fixed!